external/storm-kinesis/src/main/java/org/apache/storm/kinesis/spout/ZkInfo.java

```diff
@@ -0,0 +1,153 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.storm.kinesis.spout;
+
+import java.io.Serializable;
+
+public class ZkInfo implements Serializable {
+    // comma separated list of zk connect strings to connect to zookeeper e.g. localhost:2181
+    private final String zkUrl;
+    // zk node under which to commit the sequence number of messages. e.g. /committed_sequence_numbers
+    private final String zkNode;
+    // zk session timeout in milliseconds
+    private final Integer sessionTimeoutMs;
+    // zk connection timeout in milliseconds
+    private final Integer connectionTimeoutMs;
+    // interval at which to commit offsets to zk in milliseconds
+    private final Long commitIntervalMs;
+    // number of retry attempts for zk
+    private final Integer retryAttempts;
+    // time to sleep between retries in milliseconds
+    private final Integer retryIntervalMs;
+
+    /**
+     * Default constructor that uses defaults for a local setup
+     */
+    public ZkInfo () {
+        this("localhost:2181", "/kinesisOffsets", 20000, 15000, 10000L, 3, 2000);
```

Comment:
I think this is useful for testing but I can see users tripping over this :) so can we remove this.